.github/ISSUE_TEMPLATE/bigbluebutton-html-plugin-sdk-issue.md

```diff
@@ -0,0 +1,50 @@
+---
+name: HTML5 Issue
+about: Template for creating HTML5 Issue (frontend which you see during a session, not Greenlight).
+title: ''
+labels: 'module: client'
+assignees: ''
+
+---
+
+<!--PLEASE DO NOT FILE ISSUES FOR GENERAL SUPPORT QUESTIONS.
+This issue tracker is only for bbb development related issues.-->
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Actual behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**BBB version:**
```

Comment:
Here you likely want the SDK version too. Feel free to customize this to the plugin-sdk